pkg/agent/nodeportlocal/portcache/port_table.go

```diff
@@ -50,6 +50,14 @@ func NewPortTable(start, end int) (*PortTable, bool) {
 	return &ptable, ok
 }
 
+func (pt *PortTable) CleanupAllEntries() {
+	pt.tableLock.Lock()
+	defer pt.tableLock.Unlock()
+	for k := range pt.Table {
+		delete(pt.Table, k)
+	}
```

Comment:
while this is valid Go, why not just build a new map: `ptable.Table = make(map[int]NodePortData)`?

Reply:
Done.